Transactions/Transaction.cs

```diff
@@ -11,12 +12,27 @@ public class Transaction
         public string TransactionType { get;}
         public float Amount { get; }
         public string Title { get; }
+        public string Category { get; set; }
```

Comment:
Is Category supposed to be a free-form string? It may be an opportunity to introduce an enum here and have an enumeration of possible categories, from which the user could choose from.